software/deploy/deploy.go

```diff
@@ -427,3 +433,27 @@ func DagsOnlyDeploy(houstonClient houston.ClientInterface, appConfig *houston.Ap
 	}
 	return fileutil.UploadFile(&uploadFileArgs)
 }
+
+func UpdateDeploymentImage(houstonClient houston.ClientInterface, deploymentID, wsID, runtimeVersion, imageName string) error {
+	if runtimeVersion == "" {
+		return ErrNoRuntimeVersionPassed
+	}
```

Comment:
nit: do we need to validate that `runtimeVersion` is a valid semver and present in `updates.astronomer.io`? Or does the Houston API do that for us?